tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/TableName.java

```diff
@@ -53,6 +53,11 @@ public String getNameInDatabase()
         return nameInDatabase;
     }
 
+    public String getInternalNameInDatabase()
```

Comment:
It returns name of table without schema.
Cassandra driver requires schema to given separately.